docker/dev/requirements.txt

```diff
@@ -0,0 +1,2 @@
+diagrams
+black
```

Comment:
I don't think that we have to install the diagrams itself for local development. And we can install the black in the Dockerfile directly with `pip install black` :)

Reply:
You are right. 😅 I will change it, thanks!